cypress/integration/booleanInput.js

```diff
@@ -0,0 +1,7 @@
+describe('BooleanInput', () => {
+  it('should have a height of 35px if no description', () => {
+    cy.visit('/test/desk/booleansTest;bd99e58a-845f-4d52-b54a-56a9b7af3be1')
```

Comment:
I believe we should avoid using IDs in the URLs – just in case this document is somehow deleted by someone. How about making it so that this test first creates a new document?

Reply:
Also, maybe we should suffix test files with `.test.js` or `_spec.js` or something?

Reply:
Oh definitely! This is just an example to have a test that makes sense make it into `next`. I'll submit a fix for it quite soon after merging this 🙂